packages/bierzo-wallet/src/routes/welcome/components/ExtensionInteraction.tsx

```diff
@@ -1,3 +1,4 @@
+/*global chrome*/
```

Comment:
Can you tell me please, what is this for?

Reply:
It tells eslint that a global called `chrome` exists (https://eslint.org/docs/user-guide/configuring#specifying-globals).